src/main/resources/hudson/plugins/favorite/user/FavoriteUserProperty/config.jelly

```diff
@@ -1,33 +1,12 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
-         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
   <st:adjunct includes="hudson.plugins.favorite.assets"/>
```

Comment:
With changing this to checkboxes, the inclusion of the js is no longer necessary I think.